modules/appeals_api/spec/services/appeals_api/pdf_construction/generator_spec.rb

```diff
@@ -20,7 +20,7 @@
 
         it 'generates the expected pdf' do
           generated_pdf = described_class.new(notice_of_disagreement).generate
-          expected_pdf = fixture_filepath('expected_10182_minimum.pdf', version: 'v1')
```

Comment:
making this cap to create consistency with the other uses in the spec for slc and hlr - made a ticket to take a look at all the places where we have discrepancies in case we want to fix.